index.js

```diff
@@ -100,7 +105,9 @@ module.exports = {
       http_error: "Error de comunicación con el servidor",
       item_doesnt_exist: "El elemento no existe",
       logged_out: "Su sesión ha finalizado, vuelva a conectarse.",
-      updated: "Elemento actualizado |||| %{smart_count} elementos actualizados"
+      updated: "Elemento actualizado |||| %{smart_count} elementos actualizados",
+      i18n_error:
+        "No se pudo cargar las traducciones para el idioma especificado",
```

Comment:
Please, change by `No se pudieron cargar las traducciones para el idioma especificado`

Reply:
What is it that you want me to change. I don't understand.
Thanks.

Reply:
Hi!  
Nope, I mean to write the oration in the plural. For example: "No se pudieron..." instead of "No se pudo...". Because I think that sounds better. In` i18n_error` value.

Reply:
Ok. Done!